packages/shared/src/components/Icon.tsx

```diff
@@ -9,7 +9,8 @@ export type Size =
   | 'xlarge'
   | 'xxlarge'
   | 'xxxlarge'
-  | 'xxxxlarge';
+  | 'xxxxlarge'
+  | 'heading';
```

Comment:
Maybe rather check with Design on naming as it needs to match with their system.
Not 100% sure if the sizing is still on point.

Reply:
Tomer just responded, he reverted the icon to be at `xxxlarge` size. Let me remove this new addition quickly.